frontend/src/scenes/settings/organization/inviteLogic.ts

```diff
@@ -49,15 +49,18 @@ export const inviteLogic = kea<inviteLogicType>([
             {
                 inviteTeamMembers: async () => {
                     if (!values.canSubmit) {
-                        return { invites: [] }
```

Comment:
Amazing this didn't previously fail type validation 🤔

Reply:
We have a lot of weird type things with kea that seem to pass for now reason.